packages/vuetify/src/components/VField/VField.sass

```diff
@@ -362,10 +362,16 @@
       border-inline-start-width: var(--v-field-border-width)
 
       @include tools.ltr()
-        border-radius: $field-border-radius 0 0 $field-border-radius
```

Comment:
Was there a problem with `inherit 0 0 inherit`?

Reply:
@KaelWD Following to the document: https://developer.mozilla.org/en-US/docs/Web/CSS/border-radius
```
Note: As with any shorthand property, individual sub-properties cannot inherit, such as in border-radius:0 0 inherit inherit, which would partially override existing definitions. Instead, the individual longhand properties have to be used.
```

Reply:
Well that's lame